.github/workflows/builder.yml

```
@@ -104,11 +106,20 @@ jobs:
           password: ${{ secrets.DOCKERHUB_TOKEN }}
 
       - name: Build ${{ matrix.addon }} add-on
-        if: steps.check.outputs.buld_arch == 'true'
+        if: steps.check.outputs.build_arch == 'true'
         uses: home-assistant/builder@master
         with:
           args: |
             ${{ env.BUILD_ARGS }} \
             --${{ matrix.arch }} \
             --target /data/${{ matrix.addon }} \
-            --docker-hub homeassistant
\ No newline at end of file
+            --docker-hub homeassistant
+
+      - name: Signing image
+        if: env.BUILD_ARGS == '--docker-hub-check' && steps.check.outputs.build_arch == 'true'
+        uses: home-assistant/actions/helpers/codenotary@master
+        with:
+          source: docker://homeassistant/${{ matrix.arch }}-addon-${{ matrix.arch }}-matrix.addon:${{ steps.check.outputs.version }}
```

I think this is incorrect and should be:

```suggestion
          source: docker://homeassistant/${{ matrix.arch }}-addon-${{ matrix.addon }}:${{ steps.check.outputs.version }}
```